app/components/post_list/post/body/message/message.tsx

```diff
@@ -60,7 +60,10 @@ const Message = ({currentUser, highlight, isEdited, isPendingOrFailed, isReplyPo
     const animatedStyle = useShowMoreAnimatedStyle(height, maxHeight, open);
     const style = getStyleSheet(theme);
     const blockStyles = getMarkdownBlockStyles(theme);
-    const textStyles = getMarkdownTextStyles(theme);
+    const textStyles = useMemo(() => {
+        const _textStyles = getMarkdownTextStyles(theme);
+        return {..._textStyles, link:{..._textStyles.link, fontFamily:'Metropolis'}}
```

Comment:
Isn't this making all links (including those in the body) use `Metropolis`?

Also, I just saw the problem may be a bit more deep, since I can also reproduce the issue with something like `## heading **heading**`. Also, this one would solve it only for the post, but anywhere else markdown may be used (like post attachments) will be broken also.

I guess the issue is in part with `computeTextStyle` in `app/utils/markdown/index.ts` and in part with `getMarkdownTextStyles` in the same file.

Some juggling may be needed, but I feel in most of the places the `fontFamily` should not be there, and be inherited from the parent, but 0/5 on the details.

Reply:
Oh very good points @larkox I do agree with you.. maybe we just need to add new styles for markdown as we have `headingXText` we could do the same for `headingXLink` where `X` is the number of the heading

Reply:
confirming that this indeed does make all link text use `Metropolis`. Let me know when this is updated and I can review again @Rishy-2021

Reply:
The problem of adding a `headingLink` is that then you have to add one for each variation (take in mind that my example was no longer a heading link, but a bolded heading.

This should be solved in a more holistic way, hopefully trusting inheritance of styles.